routes/index.js

```diff
@@ -0,0 +1,5 @@
+"use strict";
+
+module.exports =(app) => {
+    app.use("/v1/user", require("./user"));
+};
```

Comment:
why don't you put that require("./user") into a variable first before passing it there. Just my own o